app/javascript/components/organizations/organization_detail.jsx

```diff
@@ -8,73 +8,90 @@ import Box from '@material-ui/core/Box'
+            <Title content={'Описание:'}/>
+          </Grid>
+          <Grid container direction="row" style={{marginTop: '10px', marginBottom: '10px'}}>
             <Grid item>
               <TypoGraphy>
                 <Box component='span' fontSize="21px" fontWeight="300">
                   {organization.description}
                 </Box>
               </TypoGraphy>
+              {organization.description === null &&
+              <TypoGraphy>
+                <Box component='span' fontSize="21px" fontWeight="300">
+                  Описание отсутсвует
+                </Box>
+              </TypoGraphy>
+              }
             </Grid>
-            <br/>
-
-
-            <Grid item xs={11} md={12}>
-              <ImageCarousel  slidesToShow={1} slidesToScroll={1} width={'100%'} height={'50%'}
-                              images={organization.images.length ?
-                                      organization.images.map((image) => image.image_url) :
-                                      "/assets/default_organization.jpg"}
-              />
-            </Grid>
-
-
-            <Grid item>
-              <TypoGraphy component="h3">
-                <Box fontSize="21px" fontWeight='300'>
-                  For each event we carry out a program! Every holiday is important to us.
-                  Nisi nisi magna deserunt tempor velit nulla ad.
+          </Grid>
+        </div>
+        <hr/>
+        
+        <div style={{marginTop: '20px', marginBottom: '20px'}}>
+          <Grid item>
+            <Title content={'Фотографии:'}/>
+          </Grid>
+          <Grid container direction="row" style={{marginTop: '10px', marginBottom: '10px'}}>
+            <Grid item xs={12} md={12}>
+              <ImageCarousel images={organization.images.map((image) => image.image_url)} slidesToShow={1}
+                             slidesToScroll={1} width={'100%'} height={'400px'}/>
+              {organization.images.length === 0 &&
+              <TypoGraphy>
+                <Box component='span' fontSize="21px" fontWeight="300">
+                  Фотографии отсутсвуют
                 </Box>
               </TypoGraphy>
+              }
             </Grid>
-
           </Grid>
-
-          <br/>
-          <br/>
-
-          <Grid container direction="row">
+        </div>
+        <hr/>
+        <div style={{marginTop: '20px', marginBottom: '20px'}}>
+          <Grid item>
+            <Title content={'Наши мероприятия:'}/>
+          </Grid>
+          <Grid container direction="row" style={{marginTop: '10px', marginBottom: '10px'}}>
             <Grid container direction="row" justify="center" alignItems="stretch">
+              {approved_events.length === 0 &&
+              <TypoGraphy>
+                <Box component='span' fontSize="21px" fontWeight="300">
+                  Мероприятия отсутсвуют
+                </Box>
+              </TypoGraphy>
+              }
               {approved_events.map((event, i) =>
```

Comment:
Can we do props destruction in this case? Example `({event: {name, start_time, end_time}, i})`